testing.sagebrush.master/Stepwat_Inputs/Input/rgroup.in

```diff
@@ -125,22 +125,11 @@ succulents		-0.0025 6.736 -0.0217 6.736
 [end]  # section end
 
 #=============================================================
-#                 Cheatgrass-Wildfire Inputs
-# These inputs allow for the implementation of wildfire based on cheatgrass biomass (g/m2)
-# This option cannot be used if prescribed fire is on. The prescribed fire and wildfire options are mutually exclusive
```

Comment:
These comments need to be retained.

Reply:
I reincorporated the first comment in commit bae2906. As for the comment "This option cannot be used if prescribed fire is on", I replaced it with
`Setting this flag to 1 will override any prescribed fire treatment(s) specified above.`
To make it very clear what will happen if you attempt to use both.